PWGHF/D2H/Tasks/taskFlowCharmHadrons.cxx

```diff
@@ -50,8 +51,9 @@ struct HfTaskFlowCharmHadrons {
   Configurable<float> zVtxMax{"zVtxMax", 10., "Max vertex coordinate z"};
   Configurable<int> harmonic{"harmonic", 2, "harmonic number"};
   Configurable<int> qvecDetector{"qvecDetector", 0, "Detector for Q vector estimation (FV0A: 0, FT0M: 1, FT0A: 2, FT0C: 3, TPC Pos: 4, TPC Neg: 5)"};
-  Configurable<int> centDetector{"centDetector", 0, "Detector for centrality estimation (V0A: 0, T0M: 1, T0A: 2, T0C: 3"};
+  Configurable<int> centDetector{"centDetector", 0, "Detector for centrality estimation (V0A: 0, T0M: 1, T0A: 2, T0C: 3)"};
   Configurable<int> selectionFlag{"selectionFlag", 1, "Selection Flag for hadron (e.g. 1 for skimming, 3 for topo. and kine., 7 for PID)"};
+  Configurable<int> nProngs{"nProngs", 3, "Number of candidate's prong (For D0, set selectionFlag = 1 and nProngs = 2)"};
```

Comment:
Thanks for the comment. 
Defining `nProng` started out as a way to differentiate between different particles for double counting in `getQvecDtracks`, but it would compile wrong, and then decaychannel was used. this can be optimized away if it really isn't necessary.
Now its only use is "float ampl = amplQVec - static_cast<float>(nProngs);"

Reply:
Of course if we need the number of prongs it is perfectly fine to have a variable, but it should not be configurable (otherwise is error prone since the user can by mistake set it to a wrong number). It is sufficient to set it at `2` for `D0` and `3` for the other channels (since you use it only one, you can also do it directly in the function where you use it)